website/docs/api/sdl/directives.mdx

```diff
@@ -197,6 +226,12 @@ the [`CommitID` scalar type](./scalars.mdx#commitid).
 
 Example: `version: CommitID! @documentVersion`.
 
+### `@immutable`
```

Comment:
Seems like a duplicate of the previous entry?

Reply:
You are right, thanks for catching this! I removed the previous entry